worker/servicefactory/worker.go

```diff
@@ -11,7 +11,8 @@ import (
 
 	"github.com/juju/juju/core/changestream"
 	coredatabase "github.com/juju/juju/core/database"
-	"github.com/juju/juju/domain/servicefactory"
+	domainsf "github.com/juju/juju/domain/servicefactory"
```

Comment:
Be consistent. In another file you made this `domainservicefactory`, which I think is better.

Reply:
Done & Done